src/pages/Editor/sections/Settings/Settings.js

```diff
@@ -33,29 +46,62 @@ const Settings = () => {
             onClick={editContext.isBodyHandler}
             className={`id-body d-none `}
           />
-          <label for="body-heading" style={{ color: !editContext.isBody && "#f0f7ff" }}>
+          <label
+            for="body-heading"
+            style={{ color: !editContext.isBody && "#f0f7ff" }}
+          >
             Body
           </label>
         </div>
       </div>
       <div className={styles.controls}>
         <div className={styles.group1}>
-          <Dropdown name="Change Color" type="color" items={["black", "red", "blue", "green", "pink"]} />
+          <Dropdown
+            name="Change Color"
+            type="color"
+            items={[
+              "black",
+              "red",
+              "orange",
+              "green",
+              "blue",
+              "dodgerblue",
+              "deeppink",
+              "darkviolet",
+            ]}
+          />
           <div className={styles.vSeparator}></div>
           <Dropdown
             name="Change Style"
             type="font"
-            items={["HomemadeApple", "Caveat", "Dawning", "IndieFlower", "NothingYouCouldDo", "Liu", "LeagueScript"]}
+            items={[
+              "HomemadeApple",
+              "Caveat",
+              "Dawning",
+              "IndieFlower",
+              "NothingYouCouldDo",
+              "Liu",
+              "LeagueScript",
+            ]}
           />
           <div className={styles.vSeparator}></div>
-          <Dropdown name="Change Sheet" type="page" items={["Ruled1", "Ruled2", "OnlyMargin", "Blank1", "Blank2"]} />
+          <Dropdown
+            name="Change Sheet"
+            type="page"
+            items={["Ruled1", "Ruled2", "OnlyMargin", "Blank1", "Blank2"]}
+          />
 
           <div className={styles.vSeparator}></div>
 
           <label className={styles.downloadBtn} htmlFor="import">
             Import File
           </label>
-          <input id="import" style={{ display: "none" }} type="file" onChange={editContext.importTxt}></input>
+          <input
+            id="import"
+            style={{ display: "none" }}
```

Comment:
Please avoid the styling of the code. If I merge this, it would cause merge conflict for all other PRs. Revert these changes. Otherwise the PR is fine!

Reply:
oh ok,
 It's got auto-formatted by Prettier.

Reply:
Code style or format issue fixed 🙂